spring-cloud-alibaba-starters/spring-cloud-starter-stream-rocketmq/src/main/java/com/alibaba/cloud/stream/binder/rocketmq/integration/RocketMQMessageHandler.java

```diff
@@ -131,7 +131,7 @@ public void start() {
 				}
 			}
 			catch (MQClientException e) {
-				logger.error("fetch publish message queues fail", e);
+				logger.error(e, "fetch publish message queues fail");
```

Comment:
What is the reason for this refactory of exeception logic?

Reply:
In the 2020 version, Spring uses LogAccessor instead of commons.logging